requirements.txt

```diff
@@ -26,3 +26,5 @@ skills-utils==0.4.0
 s3fs==0.1.5
 dill
 toolz
+tensorflow
```

Comment:
Does tensorflow have system dependencies? I'm looking at this and am unsure if we want to make tensorflow stuff available via an addon `pip install skills-ml[tensorflow]` in case it is hard to install.

Reply:
it's not that hard to install if you are not using GPU or you don't care about some low level optimization, but we can def put it to addon.